pyam/plotting.py

```diff
@@ -59,6 +59,15 @@
     'AR6-SSP4-6.0': "#C47900",
     'AR6-SSP5-3.4-OS': "#7F006E",
     'AR6-SSP5-8.5': "#990002",
+    'AR6-RCP2.6': "#003466",
```

Comment:
IMHO, I think we should separate the RCP designation with the target as is done in the SSPs above (i.e., `RCP-4.5`). Happy to hear other opinions =)

Reply:
I like consistency (people will almost certainly have their own schemes and it's easier to map from a scheme which is consistent to start with) so sounds good to me! Shall I change it here or do you want to change it in your PR?

Reply:
(On this note, should it be 'AR6-SSP3-LowNTCF' or 'AR6-SSP3-7.0-LowNTCF'? I thought it had the 7.0 forcing target applied and the lowNTCF setup applied but could be wrong..)